frontend/src/assets/delete-button.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="18" viewBox="0 0 16 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think `trash.svg` will be more better name, didn't?

Reply:
Already changed